beta/google-cloud-run/README.md

```diff
@@ -0,0 +1,187 @@
+
+    ```sh
+    gcloud services enable secretmanager.googleapis.com run.googleapis.com
+    ```
+
+3. Set the default region for Cloud Shell:
+
+    ```sh
+    gcloud config set compute/region us-central1
+    ```
+
+> [!NOTE]
+> All region-bound resources created in the following steps will be created in the specified region. You may replace `us-central1` in the above commmand with your preferred region.
+
+## Step 2: Create a secret for your `scimsession` credentials
+
+The Cloud Run service for SCIM Bridge will be configured to mount volume using a secret from Secret Manager. Follow these steps to upload your `scimsession` credentials file to the Cloud Shell, create a secret, and store the file contents as its first secret version:
+
+1. Click **⋮** _(More)_ > **Upload** in the Cloud Shell terminal menu bar.
+2. Click **Choose Files**. Select the `scimsession` file that you saved to your computer.
+3. Use the suggested destination directory. Click **Upload**.
+
+> [!NOTE]
+> If the file is saved to a different directory or using a different file name, make a note of the full path to
+> the file.
+
+4. Create a secret with the contents of this file as its first secret version using the following command:
+
+    ```sh
+    gcloud secrets create scimsession --data-file=$HOME/scimsession
+    ```
+
+> [!TIP]
+> The command above is expected work as is if the file is named `scimsession` and if it was saved to the home
+> directory when uploading the file. If not, replace `$HOME/scimsession` with the actual path to the file. For
+> example:
+>
+> ```sh
+> gcloud secrets create scimsession --data-file=/example/path/to/scimsession.file
+> ```
+
+5. Enable Cloud Run to access the secret using the Compute Engine default service account for the project:
+
+    ```sh
+    gcloud secrets add-iam-policy-binding scimsession \
+      --member=serviceAccount:$(
+        gcloud iam service-accounts list --filter="$(
+          gcloud projects describe op-scim-bridge --format="value(projectNumber)"
+        )-compute@developer.gserviceaccount.com" --format="value(email)"
+      )
+      --role=roles/secretmanager.secretAccessor
+    ```
+
+## Step 3: Deploy your SCIM Bridge
+
+Run this command to stream [`op-scim-bridge.yaml`](./op-scim-bridge.yaml) Cloud Run service YAML from this repository, use it to deploy SCIM Bridge inline, and enable public ingress for your SCIM Bridge so that you and your identity provider can connect to its public endpoint:
+
+```sh
+curl --silent --show-error \
+  https://raw.githubusercontent.com/1Password/scim-examples/solutions/pike/google-cloud-run/beta/google-cloud-run/op-scim-bridge.yaml |
+  gcloud run services replace - &&
+  gcloud run services add-iam-policy-binding op-scim-bridge --member=allUsers --role=roles/run.invoker &&
+  gcloud run services describe op-scim-bridge --format="value(status.url)"
+```
+
+<!-- !TODO: Replace the URL above with the link to its path when it is merged into main.
```

Comment:
Apply your todo so that when merged it uses the correct path.

Reply:
<!-- !TODO: do ☝🏻 before merging 🙃 -->